.gitignore

```diff
@@ -5,14 +5,14 @@ dist/
 
 # dependencies
 node_modules/
+pnpm-lock.yaml
```

Comment:
It's not really a good idea to ignore lock files, but this project uses NPM, not PNPM so it doesn't really matter too much.

When installing you should be doing so with `npm install`